master/buildbot/scripts/logwatcher.py

```diff
@@ -110,6 +110,8 @@ def start(self):
             tailBin = "/usr/xpg4/bin/tail"
         elif platform.system().lower() == 'haiku' and os.path.exists('/bin/tail'):
             tailBin = "/bin/tail"
+        elif os.path.exists('/etc/nixos'):
+            tailBin = "/run/current-system/sw/bin/tail"
```

Comment:
I think we should also check for existence of `/run/current-system/sw/bin/tail`, like in checks above.